chrome/content/zotero/itemTree.jsx

```diff
@@ -3193,6 +3196,55 @@ var ItemTree = class ItemTree extends LibraryTree {
 		if (this.collectionTreeRow.isDuplicates() && !modifierIsPressed) {
 			this.duplicateMouseSelection = true;
 		}
+
+		// Handle mouseup and mousedown events on a row in itemTree to enable clicking on +/- button
+		// On mousedown, add .active effect to the +/- button
+		// On mouseup, add/remove the clicked item from the citation
+		// This specific handling is required, since :active effect fires on the row and not the child button
```

Comment:
The :active event doesn't fire on the children because CSS `pointer-events: none` is set. See https://github.com/zotero/zotero/commit/8e2790e2d2a1d8b15efbf84935f0a80d58db4e44 . Since we don't need to support drag and drop here, we might be able to simplify the handling by enabling pointer-events. Furthermore, maybe with the current version of firefox setting `pointer-events: none` is no longer even needed.

Reply:
> In citation dialog, one can drag the row into the `bubble-input`

Huh! I didn't even realize that. Shame adding clickable cell buttons is impossible this without restoring to these kinds of workarounds.